SConstruct

```diff
@@ -446,7 +447,7 @@ consolidateObjectTable = command("consolidateObjectTable", [transformObjectCatal
                                            gen3id=patchGen3id)])
 
 # TODO : Workaround for DM-22256. forcedPhotCoadd won't be needed.
```

Comment:
Is this workaround needed any more? When we updated the conda env we got a much newer pyarrow but I don't think anyone has tried it out yet.

Reply:
I have no idea.  @hsinfang might.